sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1479,4 +1481,18 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
         |FROM (SELECT '{"f1": "value1", "f2": 12}' json, 'hello' as str) test
       """.stripMargin), Row("value1", "12", 3.14, "hello"))
   }
+
+  test ("SPARK-11633: HiveContext throws TreeNode Exception : Failed to Copy Node") {
+    val rdd1 = sparkContext.parallelize(Seq( Individual(1, 3), Individual(2, 1)))
+    val df = hiveContext.createDataFrame(rdd1)
+    df.registerTempTable("foo")
```

Comment:
Nit, most tests are now using this more concise style.  You can then avoid adding another class. 
```scala
val df = Seq((1, 3), (2, 1)).toDF("f1", "f2")
df.registerTempTable("foo")
```